### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by drasmuss**

### CODE REVIEW

File: docs/examples/spiking-fashion-mnist.ipynb [hunk 1]
```diff
@@ -186,7 +186,27 @@
...
+    "temporal dimension of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class TemporalAveragePooling(torch.nn.Module):\n",
```
Comment (drasmuss) [inline docs/examples/spiking-fashion-mnist.ipynb]: Maybe we should add this to `pytorch_spiking.modules`, since it's probably generally useful?

Comment (hunse) [reply docs/examples/spiking-fashion-mnist.ipynb]: Good idea! I'll do this

Comment (hunse) [reply docs/examples/spiking-fashion-mnist.ipynb]: Done

**Comment by hunse** [comment]
@drasmuss I think this is all good to go. The `TemporalAvgPool` module is in the last commit if you want to look at it (I changed the name slightly to match the torch pooling module naming better).

**Review by drasmuss — COMMENTED** [review]
Fixups look good!

**Comment by hunse** [comment]
Merged!

**Pull request closed without merging**